docs/examples/Tektronix_Keithley_6500.ipynb

```diff
@@ -0,0 +1,96 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# QCoDeS Example with Tektronix Keithley Digital Multimeter DMM6500"
```

Comment:
Can we add some minor info about the instrument like what it is and is used for?